lib/apruve/resources/corporate_account.rb

```diff
@@ -8,6 +8,7 @@ def self.find(merchant_id, email=nil)
       if email.nil?
         return find_all(merchant_id)
       end
+      email = URI::encode(email)
```

Comment:
`URI::encode` is deprecated and shouldn't be used any more.  Use either `CGI::escape` or `ERB::Util.url_encode` instead.  Either one of those will escape the string correctly.